frameworks/Go/chi/chi.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM techempower/go-lang:0.1
```

Comment:
FYI, we're going to be using official repos for languages now. Would you mind changing this to:

`FROM golang:1.10.1`

or whichever suits your needs. Thanks! the rest looks good.